components/Feature/TopicExplorer/index.js

```diff
@@ -4,20 +4,31 @@ const TopicExplorer = (props) => {
   const [searchTerm, setSearchTerm] = useState('')
   const [searchResults, setSearchResults] = useState([])
 
-  const onSearchChange = event => {
+  const getSearchResults = (SearchTerm) => {
```

Comment:
nit: I think parameters are camelCase in JS
```suggestion
  const getSearchResults = (searchTerm) => {
```

(I don't think we have a js linter but it's also probably not worth setting one up for a week!)

Reply:
done!